plugins/inputs/win_perf_counters/win_perf_counters.go

```diff
@@ -246,7 +252,8 @@ func (m *Win_PerfCounters) SampleConfig() string {
 }
 
 //objectName string, counter string, instance string, measurement string, include_total bool
-func (m *Win_PerfCounters) AddItem(counterPath string, objectName string, instance string, counterName string, measurement string, includeTotal bool) error {
+func (m *Win_PerfCounters) AddItem(origCounterPath string, objectName string, instance string, counterName string, measurement string, includeTotal bool) error {
+	counterPath := origCounterPath
```

Comment:
Why not
```suggestion
func (m *Win_PerfCounters) AddItem(counterPath string, objectName string, instance string, counterName string, measurement string, includeTotal bool) error {
	origCounterPath := counterPath
```
it minized the diff ;-) and is easier to follow IMO.

Reply:
Done, thanks